source/docs/tag-plugins.md

```diff
@@ -331,14 +331,50 @@ For instance:
 
 ## Include Assets
 
-Include post assets.
+Include post assets, to be used in conjunction with [`post_asset_folder`](/docs/asset-folders).
 
 ```
 {% asset_path filename %}
-{% asset_img filename [title] %}
+{% asset_img [class names] slug [width] [height] [title text [alt text]] %}
 {% asset_link filename [title] [escape] %}
 ```
 
+### Embed image
+
+"foo.jpg" is located at `http://yoursite.com/2020/01/02/hello/foo.jpg`.
+
+**Default (no option)**
+
+`{% asset_img foo.jpg %}`
```

Comment:
I cleared netlify cache to include https://github.com/hexojs/hexo/pull/4352 fix so that `{% %}` is properly escaped.